src/containers/alerts.jsx

```diff
@@ -1,23 +1,51 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 
 import {
-    showAlert,
     closeAlert
 } from '../reducers/alerts';
 
-import AlertsComponent from '../components/alerts/alerts.jsx';
+import Box from '../components/box/box.jsx';
+import Alert from '../containers/alert.jsx';
+
+const Alerts = ({
+    alertsList,
+    className,
+    onCloseAlert
+}) => (
+    <Box
```

Comment:
Since this is a container, technically it should just pass props to a component. I.e., this could get refactored in the reverse way that you refactored the behavior out of the `AlertComponent`. Not a blocker for this PR since you don't do any styling here, but would be nice eventually, maybe in the next round.